### ISSUE TITLE
Deprecate Character>>#unicodeToMacRoman after fixing the usage

### ISSUE THREAD

**Issue by MarcusDenker** [guillep/pharo-core#10872]
**Describe the problem**
 Character>>#unicodeToMacRoman was put in the Deprecated90 package, but not deprecated. It still has users (it is now in Deprecate10).

TODO
- fix users of #unicodeToMacRoman 
- Deprecate it for real with "self deprecated:" so clients can be fixed

**Comment by guillep** [comment]
Well, technically, being in the deprecated package means it's deprecated and its users should be rewritten ^^.

**Comment by svenvc** [comment]
Looking at the senders, they could be rewritten as
```
(#macroman asZnCharacterEncoder decodeBytes: { 65 }) first.
(#macroman asZnCharacterEncoder decodeAsCodePoints: { 65 }) first.

(#macroman asZnCharacterEncoder decodeBytes: { 155 }) first.
(#macroman asZnCharacterEncoder decodeAsCodePoints: { 155 }) first.
```
the input being a single byte, the output either a character or a unicode code point.

The ZnCharacterEncoders are stream based with an interface general enough to handle variable multi-byte encodings like UTF8.

Doing a byte to byte conversion can only be done for Zn[Simplified]ByteEncoders which do not have a direct method to do that. It could be added, if really necessary (i.e. that it would be performance critical).

**Comment by MarcusDenker** [comment]
The problem is that nobody knows... users do not even get a Transcript output

**Comment by guillep** [comment]
Transcript outputs are not useful, nobody looks at them. Even if the method was marked as deprecated, if it had no users nobody would have noticed. What you did is the way to go: looking at dependencies.

I think spending 10' using what @svenvc proposes is the best way to solve this issue, instead of https://github.com/pharo-project/pharo/pull/10871/files

<blockquote><img src="https://avatars.githubusercontent.com/u/413057?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/pharo-project/pharo/pull/10871">9646-unload-package-Deprecated90-in-Pharo10 by MarcusDenker · Pull Request #10871 · pharo-project/pharo</a></strong></div><div>move all methods that are in Deprecated90 package that still have users to Deprecated10

fixes  #9646 (using the workaround of just positioning the real work...)</div></blockquote>
